src/Service.Host/client/src/components/parts/Part.js

```diff
@@ -433,106 +462,94 @@ function Part({
+                                    accountingCompany={state.part.accountingCompany}
                                     handleFieldChange={handleFieldChange}
-                                    productAnalysisCode={part.productAnalysisCode}
+                                    productAnalysisCode={state.part.productAnalysisCode}
                                     productAnalysisCodeDescription={
-                                        part.productAnalysisCodeDescription
+                                        state.part.productAnalysisCodeDescription
                                     }
                                     handleProductAnalysisCodeChange={
                                         handleProductAnalysisCodeChange
                                     }
-                                    rootProduct={part.rootProduct}
-                                    department={part.department}
-                                    departmentDescription={part.departmentDescription}
-                                    handleNominalAccountChange={handleNominalAccountChange}
-                                    paretoCode={part.paretoCode}
-                                    handleAccountingCompanyChange={handleAccountingCompanyChange}
-                                    nominal={part.nominal}
-                                    nominalDescription={part.nominalDescription}
-                                    stockControlled={part.stockControlled}
-                                    safetyCriticalPart={part.safetyCriticalPart}
+                                    rootProduct={state.part.rootProduct}
+                                    department={state.part.department}
+                                    departmentDescription={state.part.departmentDescription}
+                                    paretoCode={state.part.paretoCode}
+                                    nominal={state.part.nominal}
+                                    nominalDescription={state.part.nominalDescription}
+                                    stockControlled={state.part.stockControlled}
+                                    safetyCriticalPart={state.part.safetyCriticalPart}
                                     safetyCriticalHelperText={
                                         creating() ? getSafetyCriticalHelperText() : null
                                     }
-                                    performanceCriticalPart={part.performanceCriticalPart}
-                                    emcCriticalPart={part.emcCriticalPart}
-                                    singleSourcePart={part.singleSourcePart}
-                                    cccCriticalPart={part.cccCriticalPart}
-                                    psuPart={part.psuPart}
+                                    performanceCriticalPart={state.part.performanceCriticalPart}
+                                    emcCriticalPart={state.part.emcCriticalPart}
```

Comment:
Could you destructure part from state somewhere / somehow?

Reply:
You mean just to reduce the sheer volume of props being passed into these Tab Components? I think I actually prefer really explicit props like this but it does get a bit unwieldly